CHANGELOG.md

```diff
@@ -22,6 +22,11 @@ as necessary. Empty sections will not end in the release notes.
 
 ### Commits
 
+### Note
+
+- This release has no code changes.
+- NOTICE and LICENSE files clarifications, included in jars published to Maven Central.
```

Comment:
nit: `included in published jars`... technically, they are included inside the jars, inside the docker images too.

Reply:
This mention is (only) about /NOTICE + /LICENSE